src/main/java/hek/models/EventType.java

```diff
@@ -22,15 +26,15 @@ public static EventType fromString(String s) {
         return null;
     }
 
+    /**
+     * A toString variation which has a condition for the EventType "TOB".
+     * @return String representation for EventType object
+     */
     public String toQualifiedString() {
         if(this == TOB) return "TO";
         return super.toString();
     }
 
-    public static List<EventType> getARSGFLCH() {
```

Comment:
This suppose to be already deleted? If you want to avoid conflict between master and your branch when you branch is checked out from an older master commit, you can use rebase or merge. Rebase is more convenient for me. See more detail about rebase: https://git-scm.com/docs/git-rebase